distribution/test/itests/test-itests-common/pom.xml

```diff
@@ -76,6 +76,13 @@
                     </instructions>
                 </configuration>
             </plugin>
+            <plugin>
```

Comment:
There are were no tests associated with this bundle. Since the Jacoco checks were broken, it wasn't caught. Don't think we normally run Jacoco on itest bundles though.